vms/administrator/templates/administrator/report.html

```diff
@@ -1,221 +1,40 @@
-                    </div>
-                </div>
-            </fieldset>
-            <div class="form-group">
-                <div class="col-md-12 col-md-offset-3">
-                    <button class="btn btn-primary" type="submit">{% trans "Generate Report" %}</button>
-                </div>
-            </div>
-        </form>
-    </div>
-    {% if report_list %}
-        <div class="well">
-            <legend>{% trans "Report Statistics" %}</legend>
-            <label><h4><span class="label label-default">{% trans "Total Number of Shifts" %}</span></h4></label>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp{{ report_list|length }}
-            <br>
-            <label><h4><span class="label label-default">{% trans "Total Hours" %}</span></h4></label>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp{{ total_hours }}
-        </div>
-        <div class="well">
-            <table class="table table-striped table-hover">
-                <thead>
-                    <tr>
-                        <th>{% trans "First Name" %}</th>
-                        <th>{% trans "Last Name" %}</th>
-                        <th>{% trans "Volunteer Organization"%}</th>
-                        <th>{% trans "Event Name" %}</th>
-                        <th>{% trans "Job Name" %}</th>
-                        <th>{% trans "Date" %}</th>
-                        <th>{% trans "Logged Start Time" %}</th>
-                        <th>{% trans "Logged End Time" %}</th>
-                        <th>{% trans "Duration" %}</th>
-                    </tr>
-                </thead>
-                <tbody>
-                    {% for report in report_list %}
-                        <tr>
-                            <td>{{ report.first_name }}</td>
-                            <td>{{ report.last_name }}</td>
-                            <td>{{ report.organization }}</td>
-                            <td>{{ report.event_name }}</td>
-                            <td>{{ report.job_name }}</td>
-                            <td>{{ report.date }}</td>
-                            <td>{{ report.logged_start_time }}</td>
-                            <td>{{ report.logged_end_time }}</td>
-                            <td>{{ report.duration }}</td>
-                        </tr>
-                    {% endfor %}
-                </tbody>
-            </table>
-        </div>
-    {% else %}
-        {% if notification %}
-            <div class="alert alert-danger">
-                {% trans "Your criteria did not return any results." %}
-            </div>
-        {% endif %}
-    {% endif %}
+<div class="spacer"></div>
+<h2>{% trans "Manage Reports" %}</h2>
+<hr>
+{% if report_list %}
+<table class="table table-striped table-hover">
+   <thead>
+      <tr>
+         <th>{% trans "Date Submitted" %}</th>
+         <th>{% trans "Volunteer Name" %}</th>
+         <th>{% trans "Organization" %}</th>
+         <th>{% trans "Hours" %}</th>

```

Comment:
I'll suggest to change "Hours" to "Work hours".

Reply:
It is termed as Hours in the original Report already, do you want me to change that also?